activestorage/app/controllers/concerns/active_storage/streaming.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module ActiveStorage::Streaming
+  private
+
+  # Stream the blob from storage directly to the response. The disposition can be controlled by setting +disposition+.
+  # The content type and filename is set directly from the +blob+.
+  def stream_from_storage(blob, disposition: "inline")
```

Comment:
Our documentation excludes private methods by default, so we'd need to mark it to show up:

```suggestion
  def stream_from_storage(blob, disposition: "inline") # :doc:
```